shell/styles/index.css

```diff
@@ -80,3 +80,25 @@ footer .flex-row > * {
 span.hljs-comment {
   color: #a69685 !important;
 }
+
+blockquote {
+  color: #7a7a7a;
+  font-style: normal;
+  background-color: #1f1f1f;
+  border-radius: 4px;
+  border-left: 9px solid #654ff0;
+  padding: 20px 45px 20px 26px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.15);
```

Comment:
So it seems like the block quotes have a bit or margin. Can we remove that so it lines up with the surrounding text?

![Screenshot from 2019-08-13 01-12-51](https://user-images.githubusercontent.com/1448289/62925579-7d1f5f80-bd67-11e9-8ec1-760299b0a123.png)

Reply:
will add ```blockquote { margin: 0; }```